app/gui/src/dashboard/layouts/AssetPanel/components/AssetProperties.tsx

```diff
@@ -417,6 +419,59 @@ function AssetPropertiesInternal(props: AssetPropertiesInternalProps) {
         </div>
       )}
 
+      {isSecret && isCredential && (
+        <div className={styles.section()} {...secretSpotlight.props}>
+          <Heading
```

Comment:
<Text variant="h1">{getText('configuration')}</Text>

Reply:
what does this comment mean?

Reply:
Github doesn't sanitize text bu default... fixed!